config.dev.yaml

```diff
@@ -9,7 +9,7 @@ worker_heap_limit_mb: 250
 
 # Logger info
 logging:
-  level: trace
+  level: info
```

Comment:
Let's not change the dev logging level

Reply:
Petr asked for it and I second it. The output is too noisy. Makes it hard to identify actual issues.

Reply:
Yup, I'm on the @hknustwmf side here

Reply:
I agree that in regular service development trace is too verbose, but IMHO, that's useful for someone starting out as they can see all of the data log entries provide. They can then adjust the logging level for their respective services.

Reply:
Fair enough. Changing it back.